<ui version="4.0" >
 <class>objectPZGtools</class>
 <widget class="QWidget" name="objectPZGtools" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PZG-Tools 2</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pzg.qrc" >:/new/prefix1/icons/Settings2.png</iconset>
  </property>
  <widget class="QPushButton" name="cmdExit" >
   <property name="geometry" >
    <rect>
     <x>540</x>
     <y>560</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_about" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>560</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>About PZGTools</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>560</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>v2.0rc2-Linux</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>601</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>3</number>
   </property>
   <widget class="QWidget" name="tabDatafile" >
    <attribute name="title" >
     <string>Data-file</string>
    </attribute>
    <widget class="QLineEdit" name="e_PathFile" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>471</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="b_OpenDatafile" >
     <property name="geometry" >
      <rect>
       <x>490</x>
       <y>50</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>open ...</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textEdit" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>90</y>
       <width>561</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="e_workdir" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>370</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="b_OpenWorkdir" >
     <property name="geometry" >
      <rect>
       <x>484</x>
       <y>370</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>set WorkDir</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabExtract" >
    <attribute name="title" >
     <string>Extract</string>
    </attribute>
    <widget class="QCheckBox" name="chk_03" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>60</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>0.3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Select particle size you want to extract (particle size in Âµm):</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_05" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>90</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>0.5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_07" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>120</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>0.7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_10" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>150</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>1.0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_20" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>60</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>2.0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_50" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>90</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>5.0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_70" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>120</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>7.0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_100" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>150</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>10.0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_extract" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>300</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Extract </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="e_outfile" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>230</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>out.dat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>210</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Name of the output data-file prefix:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPlot" >
    <attribute name="title" >
     <string>Plotting</string>
    </attribute>
    <widget class="QLineEdit" name="e_plotfile" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>40</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>myplot.plt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="e_title" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Do not use spaces for the title name, use _ or - , for example: laminarflow_0.3um_roomnumber_346</string>
     </property>
     <property name="text" >
      <string>title_of_the_plot</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>160</y>
       <width>541</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title" >
      <string>Select data-files from extraction:</string>
     </property>
     <widget class="QPushButton" name="b_openData1" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>Data-file 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_openData2" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>Data-file 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_openData3" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>110</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>Data-file 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_openData4" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>150</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>Data-file 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_openData5" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>190</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>Data-file 5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_data1" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>30</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_data2" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>70</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_data3" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>110</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_data4" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>150</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="e_data5" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>190</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="b_plot" >
      <property name="geometry" >
       <rect>
        <x>440</x>
        <y>270</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>Plot!</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>20</y>
       <width>231</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>Name of the plotfile to create:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>80</y>
       <width>261</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>Title of the plot (do NOT use spaces!) :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabClimate" >
    <attribute name="title" >
     <string>Climate data</string>
    </attribute>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>30</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Select climate data you want to plot:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_getClima" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>280</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Show</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="e_climateWorkDir" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>230</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_temp" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>60</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_humi" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>90</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>Humidity</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_pres" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>120</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>Pressure</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_velo" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>150</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text" >
      <string>Gas-Velocity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>210</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Working-directory:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pzg.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cmdExit</sender>
   <signal>clicked()</signal>
   <receiver>objectPZGtools</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>549</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel" >
     <x>459</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
